pkgs/development/skaware-packages/tipidee/default.nix

```
@@ -0,0 +1,40 @@
+{ skawarePackages }:
+
+with skawarePackages;
+
+buildPackage {
+  pname = "tipidee";
+  version = "0.0.1.0";
+  sha256 = "sha256-rKi9IX9CcRhY4n44i2vDom9MIeuGxRAHF7u0C3nNvFU=";
```

```suggestion
  hash = "sha256-rKi9IX9CcRhY4n44i2vDom9MIeuGxRAHF7u0C3nNvFU=";
```

---

The `buildPackage` function here does not currently take a `hash` argument.

But this would be possible:

```suggestion
  sha256 = "rKi9IX9CcRhY4n44i2vDom9MIeuGxRAHF7u0C3nNvFU=";
```

---

Sounds good to me (: